hs_discover/views.py

```diff
@@ -260,12 +280,8 @@ def get(self, request, *args, **kwargs):
                 "geo": pt
             })
 
-        if sort == 'title':
-            resources = sorted(resources, key=lambda idx: idx['title'].lower())
-        elif sort == '-title':
-            resources = sorted(resources, key=lambda idx: idx['title'].lower(), reverse=True)
-
         return JsonResponse({
+            "filterdata": json.dumps(filterdata),
             'resources': json.dumps(resources),
```

Comment:
Every call will include facet counts for redrawing integers next to checkboxes as resources are filtered by map mode, date, freetext search or filtering checkbox actions.

Reply:
By including filterdata on each response, you are addressing #3998

Reply:
Yes*

Reply:
There is an addition here, which stakeholders and not we will weigh in on